GoogleForCreators/web-stories-wp
# Pagination for saved page templates

## Summary
Adds pagination to the page templates so that 100 would be loaded with each page.


## Relevant Technical Choices



## To-do



## User-facing changes
100 page templates are now loaded instead of all, when scrolling down, more page templates are loaded while scrolling.


## Testing Instructions
- Verify you have at least 100 page templates, or alternatively change the [`per_page`](https://github.com/google/web-stories-wp/pull/7477/files#diff-777c3aac9f73b7ce2a33259909beecc1be11988fbfb937c5059a4a9620b4f7d9R393) to a smaller value for the time of testing.
- Verify that the next pages are loaded while scrolling down.
- Verify that once scrolled to the end, no more requests are made to get more pages.
- Leave the Templates panel, go back. Verify that all the page templates are still loaded and no more requests are happening.


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #7271
